app/views/users/profile.html.erb

```diff
@@ -296,6 +298,48 @@
       $("#social-links").toggle();
   });
 
+  $(document).ready(function() {
+    function splitText(text) {
+        var textBreak = textLimit;
```

Comment:
As per the code, there isn't a need for such a wide scoped variable declaration using `var`, it will be good to use `let` here.